apps/src/studio/studio.js

```diff
@@ -2108,12 +2146,28 @@ Studio.reset = function(first) {
     .setAttribute('visibility', 'hidden');
   document.getElementById('victoryText')
     .setAttribute('visibility', 'hidden');
-  var resetText = document.getElementById('resetText');
-  if (level.tapSvgToRunAndReset) {
-    resetText.textContent = studioMsg.tapOrClickToPlay();
-    resetText.setAttribute('visibility', 'visible');
+  if (dom.isMobile() || dom.isWindowsTouch()) {
```

Comment:
This looks like it maight show up in other games which use studio. Can you please verify that this doesn't show up unexpectedly in `/s/playlab/stage/1/puzzle/1` (playlab) or `/s/algebra/stage/7/puzzle/1` (big game)?